mcs/class/System/Test/System.Net/DnsTest.cs

```diff
@@ -205,12 +206,21 @@ public void GetHostByName ()
 			}
 		}
 
+		static IPAddress GetNonIPv6Address (IPHostEntry h)
+		{
+			var al0 = h.AddressList [0];

```

Comment:
Can we do `h.AddressList.FirstOrDefault(x => x.AddressFamily == AddressFamily.InterNetwork)` so this test would still work in a mixed ipv6/ipv4 network where a v6 address is the first in the list?